docs/self-hosted-appcircle/configure-server/_ufw-configuration.mdx

```diff
@@ -0,0 +1,20 @@
+If you are using `UFW (Uncomplicated Firewall)`, you need to open the 80 and 443 ports for the Appcircle server.
```

Comment:
In DMZ docs, it should be Appcircle DMZ server.

<img width="842" alt="Screenshot 2024-07-10 at 12 46 29" src="https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/c257a356-6f05-4b6b-8524-61e48af1db31">

Or, we can remove these dangerous words on re-used document parts. Or, ee can replace them with more generic words that  will mention the relevant component for both referenced docuemnts.

Reply:
🆗